signing/Makefile

```diff
@@ -0,0 +1,26 @@
+include ./bbpass
```

Comment:
Changed it to execute the commands instead of echoing them. Sorry I can't test in bash, anything else?